woodwork/column_accessor.py

```diff
@@ -67,6 +68,49 @@ def description(self, description):
         _validate_description(description)
         self._schema['description'] = description
 
+    @property
+    def iloc(self):
+        """
+        Integer-location based indexing for selection by position.
+        ``.iloc[]`` is primarily integer position based (from ``0`` to
+        ``length-1`` of the axis), but may also be used with a boolean array.
+
+        Allowed inputs are:
+            An integer, e.g. ``5``.
+            A list or array of integers, e.g. ``[4, 3, 0]``.
+            A slice object with ints, e.g. ``1:7``.
+            A boolean array.
+            A ``callable`` function with one argument (the calling Series, DataFrame
+            or Panel) and that returns valid output for indexing (one of the above).
+            This is useful in method chains, when you don't have a reference to the
+            calling object, but would like to base your selection on some value.
+        """
+        return _iLocIndexerAccessor(self._series)
+
+    @property
+    def loc(self):
+        """
+        Access a group of rows and columns by label(s) or a boolean array.
+        ``.loc[]`` is primarily label based, but may also be used with a
+        boolean array.
+        Allowed inputs are:
+        - A single label, e.g. ``5`` or ``'a'``, (note that ``5`` is
```

Comment:
Seems like the docs are failing maybe because of the indentation here? We should probably have the format match `iloc`

Reply:
Should be fixed now. I hope.